833 replace core extension language (#1193)
open-contracting/standard: docs/guidance/map/extensions.md

OCDS provides a common core of [sections and building blocks](../../getting_started/building_blocks) for describing contracting processes.

Many publishers will have additional data that they could publish. Instead of ignoring this data and leaving it unpublished, OCDS encourages publishers to collaborate on the creation of **extensions** to the standard.

> Some extensions are maintained by the Open Contracting Partnership (OCP) as part of the [standard governance process](../../governance/index), documented as part of the standard, and reviewed by the OCDS technical team with each version upgrade of OCDS. They are likely to be relevant to a large number of publishers and users.

Reviewer: Extensions are no longer documented as part of the standard - we instead link to the Extension Explorer.

The following is implied by "as part of the governance process": "reviewed by the OCDS technical team with each version upgrade of OCDS"

The first sentence left it ambiguous whether all or some of OCP's extensions followed the governance process.

I moved a later sentence here so that it appeared in context.

The governance page now uses the phrase "standard development team"

```suggestion
Some extensions are maintained by the Open Contracting Partnership (OCP). Those that are likely to be relevant to a large number of publishers and users follow the [governance process](../../governance/index). The standard development team decides which extensions follow the governance process.
```

> Other extensions are maintained outside of the standard governance process, or are maintained by third parties such as the OCDS community or an OCDS publisher. These extensions might provide features which are needed by only a small number of publishers or users, or they might be used to document a specific publisher's additional fields or codelist values.

Reviewer: ```suggestion
Other extensions are maintained by third parties, like OCDS publishers, and do not follow the governance process. These extensions might provide features which are needed by only a small number of publishers or users, or they might be used to document a specific publisher's additional fields or codelist values.
```

> All registered extensions are listed in the [Extension Explorer](https://extensions.open-contracting.org/en/). The [standard technical team](../../governance/index) approves extensions for inclusion in the Extension Explorer (distinguishing community from local extensions) and for inclusion in the governance process.

Reviewer: I moved a sentence up so that it appeared in context.

We don't use "registered" earlier, so I added a clause to clarify.

The governance page now uses the phrase "standard development team".

```suggestion
All OCP extensions and many third-party extensions are documented in the [Extension Explorer](https://extensions.open-contracting.org/en/). The [standard development team](../../governance/index) decides which extensions are registered in the [Extension Registry](https://github.com/open-contracting/extension_registry#readme), which controls which extensions appear in the Extension Explorer.
```

## Using existing extensions

Extensions are applied by adding their URLs to the `extensions` array in the release package or record package. You can discover extensions, read their documentation and find the URL to add using the [Extension Explorer](https://extensions.open-contracting.org/en/).

> This version of OCDS uses these specific versions of extensions:

Reviewer: ```suggestion
This version of OCDS uses these specific versions of the following extensions:
```
